facturacion-web/src/app/landing/menu-panel/menu-panel.component.html

```diff
@@ -1,14 +1,40 @@
 <div class="panel-container">
+  <div class="sidebar-header">
+    <h5>FacturacionWeb</h5>
+  </div>
+  <div class="sidebar-profile">
+    <div class="user-pic">
+      <img class="img-responsive img-rounded" src="assets/images/logo.png" alt="User picture">
```

Comment:
```suggestion
      <img class="logo-img" src="assets/images/logo.png" alt="User picture">
```
y agrupar los estilos en una sola clase